bot/backend/apis/music.py

```diff
@@ -0,0 +1,24 @@
+"""interacting with the music api"""
+
+import tekore
+
+from decouple import config
+
+
+class MusicClient:
+    """Manage music searches and recommendations"""
+
+    def __init__(self):
+        client_id = config("SPOTIFY_CLIENT_ID")
```

Comment:
Imo it's good practice to add some notes about newly added env vars to the PR description (moreover in our case, we have a .env example in the README as well that needs to be updated)

Reply:
I'll do it soon